### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by drakus72**
made a note in docker section to change from bridge mode to host mode in docker for tv guide data to work

### CODE REVIEW

**Comment by nvllsvm** [comment]
I'm assuming this works for you, but I'd like to understand why before this gets merged.

File: README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ Prebuild packages are available for Debian/Ubuntu and Arch, and via Docker Hub.
 
 The Jellyfin Docker image is available on Docker Hub at https://hub.docker.com/r/jellyfin/jellyfin/
 
+Set to **mode host & not bridge** for the TV guide to work proeprly
```
Comment (BnMcG) [inline README.md]: Typo: 'properly'.

**Comment by dali99** [comment]
I run emby with port publishing, (-p) so I doubt this is needed at all

**Comment by joshuaboniface** [comment]
Yea, I think port publishing is the usual best practice here is it not? This is a pretty major default network change, and if there's a "single best way" to solve it that might be better.

However, I have to reject this PR anyways because we've move all of this sort of documentation to the Wiki. I can make a mention of it there however. @jaredallard @dall99 can you confirm what would be needed exactly so we can document that?

**Comment by nvllsvm** [comment]
Closing - need a reason why host networking is required instead of using Docker port facilities.

**Pull request closed without merging**